data/lang/module-searchrescue/en.json

```diff
@@ -694,5 +694,14 @@
   "WHERE_IS_THE_TARGET": {
     "description": "",
     "message": "Where do I find the target?"
+  },
+  "PLAYER_DESTROYED_TARGET": {
+    "description": "",
+    "message": "We have heard of your deeds, captain! Rest assured that we will report you to the local authorities and get you flagged for completely unacceptable behavior. A criminal lawsuit will follow."
```

Comment:
Rather than " get you flagged for completely unacceptable behavior", it might be good if it specified why, e.g. "for destroying the very ship you were sent out to aid" or similar.